src/wp-includes/class-wp-theme-json.php

```diff
@@ -2013,4 +2073,152 @@ public static function get_from_editor_settings( $settings ) {
 		return $theme_settings;
 	}
 
+	/**
+	 * Returns the current theme's wanted patterns(slugs) to be
+	 * registered from Pattern Directory.
+	 *
+	 * @since 6.0.0
+	 *
+	 * @return array
+	 */
+	public function get_patterns() {
+		if ( isset( $this->theme_json['patterns'] ) && is_array( $this->theme_json['patterns'] ) ) {
+			return $this->theme_json['patterns'];
+		}
+		return array();
+	}
+
+	/**
+	 * Returns a valid theme.json as provided by a theme.
+	 *
+	 * Unlike get_raw_data() this returns the presets flattened,
+	 * as provided by a theme. This also uses appearanceTools
+	 * instead of their opt-ins if all of them are true.
+	 *
```

Comment:
This is wrapping at a line length much less than is specified in our [code style guide](https://developer.wordpress.org/coding-standards/inline-documentation-standards/php/#line-wrapping).

Reply:
Updated in ca12de485b